packages/um/package.py

```diff
@@ -225,6 +233,44 @@ def check_model_vs_spec(model, config_env, var, spec_value):
                         f"The value {spec_value} will be used.")
 
 
+        def check_model_vs_sources_vs_ref(
+            model,
+            config_env,
+            sources_var,
+            ref_var,
+            resource_path):
+            """
+            Check the values set by the variants sources_var and ref_var
+            against any existing sources value in config_env, and remind
+            that the sources value will be overridden by resource_path.
+            """
+            sources_value = spec.variants[sources_var].value
+            ref_value = spec.variants[ref_var].value
+            tty.info(f"The spec sets {ref_var}={ref_value}")
+            if sources_value == "none":
+                # In this case, the spec has not overridden any value
```

Comment:
"any value"? Or has not overridden `sources_var`'s value?

Reply:
I changed the comment. Hopefully it is clearer.